### PR TITLE
Update the Readme instructions

### PR DESCRIPTION
**Pull request by lemirep**
To explain how to install the addons from pre generated zip packages

Task-Id: KUE-589

### CODE REVIEW

**Comment by TimoBuskeKdab** [comment]
Not sure if this is still a problem but earlier days I sometimes had problems using our exporter when I already had installed the original Khoros exporter and loading them side by side (maybe namespace pollution? maybe even depending on the order the exporters get loaded?). We could mention to disable other gltf exporters before enabling ours if the user encounters problems.

**Comment by lemirep** [comment]
Good point, will mention that

File: README.md [hunk 1]
```diff
@@ -22,7 +22,30 @@ The Kuesa exporter and tools for Blender requires Blender 2.78 or later
...
+2. Go to: Files -> User Preferences -> Add-ons
+3. In the Add-ons search field write down gltf
+4. Deactivate all active entries
+
+### Instructions if you just want to use the addons from prebuilt packages
+
+1. Download the zip files for each addon from https://bit.ly/2TabBkD
+   * kuesa_latest.zip
+   * kuesa_exporter_latest.zip
+2. Fire up Blender
```
Comment (mkrus) [inline README.md]: Launch Blender

File: README.md [hunk 2]
```diff
@@ -22,7 +22,30 @@ The Kuesa exporter and tools for Blender requires Blender 2.78 or later
...
+3. Go to: Files -> User Preferences -> Add-ons -> Install Add-on from File...
+4. Search and add each zip file (one at a time)
+5. In the Add-ons search field write down kuesa
```
Comment (mkrus) [inline README.md]: In the Add-ons search field,  type `kuesa`

**Comment by lemirep** [comment]
Updated bitly url, good call

**Pull request merged**